ts/WoltLabSuite/Core/Bootstrap.ts

```diff
@@ -30,9 +30,11 @@ import * as UiEmpty from "./Ui/Empty";
 import * as UiObjectAction from "./Ui/Object/Action";
 import * as UiObjectActionDelete from "./Ui/Object/Action/Delete";
 import * as UiObjectActionToggle from "./Ui/Object/Action/Toggle";
+import { init as initSearch } from "./Ui/Search";
 
 // perfectScrollbar does not need to be bound anywhere, it just has to be loaded for WCF.js
 import "perfect-scrollbar";
+import { PageMenuMainProvider } from "./Ui/Page/Menu/Main/Provider";
```

Comment:
This is an odd placement for this `import`. I'd move this up above perfect scrollbar.

Reply:
Thanks, it was implicitly added by the IDE.